src/pages/markets/components/helper/_available-platforms.js

```diff
@@ -64,12 +64,12 @@ const AvailablePlatforms = ({
     derivx,
     flex_direction,
     tablet_direction,
-    }) => {
+}) => {
     const { is_eu_country } = React.useContext(DerivStore)
 
     return (
         <Flex
-            style={{'flexWrap': 'wrap'}}
```

Comment:
I know this isn't the scope but just can't help noticing this. Let's remove this and use the `wrap` prop instead to pass in the wrap property for `Flex`.

Reply:
> I know this isn't the scope but just can't help noticing this. Let's remove this and use the `wrap` prop instead to pass in the wrap property for `Flex`.

OK => done in the 4th commit =)